samples/external/github.yaml

```diff
@@ -17,24 +17,6 @@ spec:
     name: https
     protocol: HTTPS
 ---
```

Comment:
remove this too?

Reply:
Testing with release-1.1-20190214-09-16 showed this was needed.